cmd/export_ledger_entry_changes.go

```diff
@@ -43,18 +48,13 @@ be exported.`,
 			cmdLogger.Fatal("could not get absolute filepath for the config file: ", err)
 		}
 
-		/*var outFile *os.File
-		if !useStdout {
-			outFile = mustOutFile(path)
-		}*/
-
 		core, err := input.PrepareCaptiveCore(execPath, configPath, startNum, endNum)
 		if err != nil {
 			cmdLogger.Fatal("error creating a prepared captive core instance: ", err)
 		}
 
 		accChannel, offChannel, trustChannel := createChangeChannels(exportAccounts, exportOffers, exportTrustlines)
-		go input.StreamChanges(core, startNum, endNum, accChannel, offChannel, trustChannel)
+		input.StreamChanges(core, startNum, endNum, accChannel, offChannel, trustChannel)
```

Comment:
This function needs to be run in the background as a goroutine, otherwise it'll be blocking, and you'll never get to the below code. The `go` is necessary to kick that off.

Reply:
Ah, I see, you made the function run a goroutine. I'm not sure that'll prevent blocking, but I can believe it works. In either case, I think it's better to kick off the goroutine in the CLI - it'll be clearer and more readable.